dictionary/lib/dictionary/agent.ex

```diff
@@ -0,0 +1,16 @@
+defmodule Dictionary.Agent do
+  alias Dictionary.WordList, as: WordList
+
+  @me DICTIONARYagent
+
+  def start_link() do
+    Agent.start_link(fn -> WordList.word_list end, name: @me)
+  end                                                           # End of start_link
+
+
+  def random_word() do
```

Comment:
No deduction, just a note: you could also write this as

~~~ elixir
Agent.get(@me, &WordList.random_word/1)
~~~